### DISCUSSION TITLE
Not working on IMG2IMG

### DISCUSSION THREAD

**Discussion by LucianoDaluz** [glucauze/sd-webui-faceswaplab#193] [General]
Getting this error on Image2iMAGE: 
 Failed to swap face in postprocess method : apply_overlay() takes 3 positional arguments but 4 were given

Does anybody knows how to fix it? Thanks

**Comment by Ermak12345** [comment]
Forgive my English but this is how i fix the img2img

1 - generate an image in text to image using faceswap.
2 - send it to inpaint in the options below the generated image.
3 - mask the face in inpaint and use faceswap, generate.
4 - after the inpaint generation, use img2img normally, using faceswap.

Don't need to do the steps all over again, unless you close stable diffusion

**Reply by LucianoDaluz** [reply]
Thanks, but it didnt work, Still getting the message, and no swap:  FaceSwapLab - ERROR - Failed to swap face in postprocess method : apply_overlay() takes 3 positional arguments but 4 were given

**Reply by Ermak12345** [reply]
Forgot to tell you, you need to chose "Only masked" on the inpaint area option before generation.
